webapp/src/components/modals/channel_settings/select_channel_subscription.tsx

```diff
@@ -92,7 +108,7 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
                                 {' - '}
                                 <button
                                     className='style--none color--link'
-                                    onClick={this.handleDeleteChannelSubscription}
+                                    onClick={(): void => this.handleDeleteChannelSubscription(sub)}
```

Comment:
You are supposed to avoid lambda functions inline in JSX because it makes a new one every render.

Reply:
This one is in a loop, so I'm not sure what is the best way to avoid it.

Something like `<button id={sub.id} />`

then the `onClick` would take the click event and get the subscription id from `e.target.id`. @crspeller What do you think?

Reply:
Actually forget it. I don't think it's that big a deal and that solution just makes the code cryptic. This is clear.